docs/examples.rst

```diff
@@ -263,11 +263,11 @@ CybOX type.
 
     <cyboxCommon:HashListType>
         <cyboxCommon:Hash>
-            <cyboxCommon:Type xsi:type="cyboxVocabs:HashNameVocab-1.0">MD5</cyboxCommon:Type>
+            <cyboxCommon:Type>MD5</cyboxCommon:Type>
```

Comment:
Oh, I see. Maybe `from_dict` should see if there's a matching value in the "default" vocabulary and if so, use the correct xsi:type? That would simplify the dictionary representation a lot (if only we could do the same with objects!)